internal/events/event_dispatcher.go

```diff
@@ -159,6 +159,10 @@ func (ed *eventDispatcher) getEvents(ctx context.Context, filter database.Filter
 }
 
 func (ed *eventDispatcher) enrichEvents(events []fftypes.LocallySequenced) ([]*fftypes.EventDelivery, error) {
+
+	// TODO: Update this to use the message cache, only for event types that have message references.
+	//       Do this after Alex's changes to enrichment are merged
```

Comment:
I think these changes are merged, but unsure if this TODO was meant to be here or in a follow-up.

Reply:
The TODO has been addressed in #593